spec/spec_helper.rb

```diff
@@ -1,8 +1,8 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
Great! Thank you so much for catching this and working extra to figure this out!